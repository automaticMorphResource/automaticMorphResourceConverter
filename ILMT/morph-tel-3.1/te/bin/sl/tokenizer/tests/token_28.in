వీరి వంశంలో 8వ తరానికి చెందిన రెండవ మాదానాయుడు కాలంలోనే దేవరకొండ దుర్గం నిర్మాణం జరిపినట్లు చరిత్రకారులు చెబుతున్నారు . 
