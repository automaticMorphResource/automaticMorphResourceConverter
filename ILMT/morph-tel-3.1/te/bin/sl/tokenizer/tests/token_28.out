<Sentence id="1">
1	వీరి	unk
2	వంశంలో	unk
3	8వ	unk
4	తరానికి	unk
5	చెందిన	unk
6	రెండవ	unk
7	మాదానాయుడు	unk
8	కాలంలోనే	unk
9	దేవరకొండ	unk
10	దుర్గం	unk
11	నిర్మాణం	unk
12	జరిపినట్లు	unk
13	చరిత్రకారులు	unk
14	చెబుతున్నారు	unk
15	.	unk
</Sentence>

